fix: problems with getting enviroment variables from parent folder s… (#483)
NordicSemiconductor/asset-tracker-cloud-docs: docs/aws/GettingStarted/Webapp.rst

       echo "export PUBLIC_VERSION=\"`git describe --tags $(git rev-list --tags --max-count=1)`\"" >> .envrc

> #. To get the variables from the :file:`.envrc` file in the parent folder (:file:`nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:

Reviewer: ```suggestion
#. To get the variables from the :file:`.envrc` file in the parent folder (:file:`~/nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:
```

   .. code-block:: bash

      source_up

> 

Reviewer 2: Just a nit: You might want to remove an extra empty line here.

       echo "export PUBLIC_VERSION=\"`git describe --tags $(git rev-list --tags --max-count=1)`\"" >> .envrc

> #. In order to get the variables from the :file:`.envrc` file in our parent folder (:file:`nrf-asset-tracker`) we also need to add the following:

Reviewer 2: ```suggestion
#. To get the variables from the :file:`.envrc` file in the parent folder (:file:`nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:
```

   .. code-block:: bash

      source_up

>    in the :file:`.envrc` in our current folder, :file:`web-app`. 

Reviewer 2: ```suggestion
```
